native/src/main/java/io/ballerina/persist/objects/GenerateBalException.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Rename to `BalExceptions`

Reply:
Updated in [6d8b4a4](https://github.com/ballerina-platform/persist-tools/pull/15/commits/6d8b4a41b9a0ea1002bb906bb2008380cfa7cdb1)